app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -273,3 +291,12 @@ private fun getFakePocketStories(limit: Int = 1): List<PocketRecommendedStory> {
         }
     }
 }
+
+private class FakeClient : Client() {
+    override fun fetch(request: Request) = Response(
+        url = request.url,
+        status = 200,
+        body = Response.Body.empty(),
+        headers = MutableHeaders()
+    )
+}
```

Comment:
I wonder how we can keep track of those fakes, or even provide some standard implementations. For one it would be nice to avoid having multiple of those implementations everywhere, and second, it feels a bit like they pollute the code base, or distract from the actual runtime code.

Reply:
I don't know of an idiomatic way to achieve this.
Only an ugly way of having a new package - `fakes`, maybe inside `composables` since these would be used in multiple Composables previews?

Reply:
Maybe we can even ship some as AC package. I was recently thinking about something similar for tests, sometimes some good fakes are nicer than mocking manually.